### PR TITLE
Prepare setup.py for official release

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun <kaplun@protonmail.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,10 +41,10 @@
     description="Unofficial Linux GUI client for ProtonVPN",
```
Comment (calexandru2018) [inline setup.py]: Missed a spot. Btw should we add the readme to the pypi docs or just re-route to Github ?

Comment (kaplun) [reply setup.py]: It would be nice if the Readme appears on Pypi indeed.

Comment (kaplun) [reply setup.py]: Can I leave the adding of README to setup.py to you? I am no-longer fresh on setup.py :-)

Comment (calexandru2018) [reply setup.py]: @kaplun It is already added. If you check the pypi page for the project, you'll see the readme. I was just making sure if we should have it or not.

**Pull request merged**